### PR TITLE
Fixes #22395 - Added DEFAULT_REFRESH_INTERVAL constant

### PR DESCRIPTION
**Pull request by witlessbird**

### CODE REVIEW

File: lib/foreman_remote_execution_core/fake_script_runner.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module ForemanRemoteExecutionCore
   class FakeScriptRunner < ForemanTasksCore::Runner::Base
-
+    DEFAULT_REFRESH_INTERVAL = 60
```
Comment (adamruzicka) [inline lib/foreman_remote_execution_core/fake_script_runner.rb]: Shouldn't the `FakeScriptRunner` use the same default interval as a regular `ScriptRunner`, i.e. 1 second?

Comment (iNecas) [reply lib/foreman_remote_execution_core/fake_script_runner.rb]: +1 on 1 second

**Comment by witlessbird** [comment]
* changed the default interval to 1 second.

**Comment by iNecas** [comment]
Thanks @witlessbird

**Pull request merged**